### ISSUE TITLE
[Feature Request] Support Windows 10 light theme

### ISSUE THREAD

**Issue by jordi-7** [White-Tiger/T-Clock#162]
Windows 10 introduced a system light theme that tints the taskbar in the latest insider previews, which will come to the stable version in the first half of 2019.
https://blogs.windows.com/windowsexperience/2018/11/14/announcing-windows-10-insider-preview-build-18282/#BB7iwCs7YmIQzLBe.97
Right now, going into the settings of T-clock is required to change the clock's color every time the system theme changes. It would be great that T-clock could change automatically.

**Comment by White-Tiger** [comment]
well... if T-Clock really doesn't work in this situation _(because it should)_ it's either an issue detecting theme changes, or that Microsoft isn't using the "normal" way to change themes anymore...
Right now, T-Clock is using the current theme's colors to draw itself _(I tried to make it draw the same way the stock clock does)_ and it works most of the time... _(eg. high contrast themes aren't working 100% right)_

But I'll have a look at some point in the future.

**Comment by jordi-7** [comment]
Great to hear you'll look at it.
For me at least it doesn't work when changing the theme from settings. I use the task scheduler to change themes based on the time of the day. The task modifies the registry to change themes. There are to values that need to be changed under HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize: AppsUseLightTheme and SystemUsesLightTheme (this one tints the taskbar). Maybe that's the issue? In previus versions you could only change the theme for UWP apps, not the system.

**Comment by White-Tiger** [comment]
yeah, it doesn't sound like themes in the "classic" way, so will probably require some special handling and dirty hacks^^ _(I hope it doesn't involve hacks, but I fear it does)_

**Comment by jordi-7** [comment]
Any news?

**Comment by morales-ivan** [comment]
Hey I'd like to bump this issue. I have the latest pre-release version and the text color still doesn't adjust accordingly to the system theme. I'm running Windows 10 18363.720 (1909)

**Comment by morales-ivan** [comment]
I'd like to add and say that, even a feature where one could set two times and the text colors to change to would be nice, so we can manueally sync it with the alternating theme.

**Comment by benjamin051000** [comment]
Bumping this! I use [Auto Dark Mode](https://github.com/Armin2208/Windows-Auto-Night-Mode) That switches my system theme from Light to Dark at night (and back to Light in the morning). Each time the theme switches, I have to change my T-Clock color manually each time this happens. Some automation here would be AMAZING. I love this app!

**Comment by pinkcrowberry** [comment]
Bumping this, just to show there is still interest in this ^^ Similar situation as others, I use an automatic dark theme switcher, and the clock colour doesn't change automatically. Even just having a way to do it automatically with set times would help immensely!

**Comment by DiskuSystem** [comment]
Hey man, just add a scheduler for the theming in T-Clock and this will all be fixed. 

Is the developer dead?
